include/libaktualizr/packagemanagerinterface.h

```diff
@@ -54,6 +54,7 @@ class PackageManagerInterface {
   virtual void completeInstall() const { throw std::runtime_error("Unimplemented"); }
   virtual data::InstallationResult finalizeInstall(const Uptane::Target& target) = 0;
   virtual void updateNotify() {}
+  virtual void installNotify(const Uptane::Target& target) {}
```

Comment:
`
/home/mike/work/foundries/projects/aklite-boot/aktualizr/include/libaktualizr/packagemanagerinterface.h: In member function ‘virtual void PackageManagerInterface::installNotify(const Uptane::Target&)’:
/home/mike/work/foundries/projects/aklite-boot/aktualizr/include/libaktualizr/packagemanagerinterface.h:57:52: error: unused parameter ‘target’ [-Werror=unused-parameter]
   virtual void installNotify(const Uptane::Target& target) {}
                                                    ^~~~~~
cc1plus: all warnings being treated as errors

`